startup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Start script for chips rest interfaces consumer
+
+export APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PROXY_ARGS=""
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+
+    source ~/.chs_env/private_env
+    source ~/.chs_env/global_env
+    source ~/.chs_env/chips-rest-interfaces-consumer/env
+
+    PORT="${CHIPS_REST_INTERFACES_CONSUMER_PORT:= }"
+
+else
+
+    PORT="$1"
+    CONFIG_URL="$2"
+    ENVIRONMENT="$3"
+    APP_NAME="$4"
+
+    source /etc/profile
+
+    echo "Downloading environment from: ${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}"
+    wget -O "${APP_DIR}/private_env" "${CONFIG_URL}/${ENVIRONMENT}/private_env"
+    wget -O "${APP_DIR}/global_env" "${CONFIG_URL}/${ENVIRONMENT}/global_env"
+    wget -O "${APP_DIR}/app_env" "${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}/env"
+    source "${APP_DIR}/private_env"
+    source "${APP_DIR}/global_env"
+    source "${APP_DIR}/app_env"
+
+fi
+
+exec java ${JAVA_MEM_ARGS} -jar ${PROXY_ARGS} -Dserver.port="${PORT}" -Dspring.data.mongodb.uri="$MONGODB_URL" "${APP_DIR}/chips-rest-interfaces-consumer.jar"
```

Comment:
I'd remove the -Dspring.data.mongodb.uri="$MONGODB_URL" as currently the service isn't interacting with mongo. Plus we should probably set this in the application.properties file so it is more visible

Reply:
Mongo removed - keeping the rest of the line as it is for now.